express-get-json/index.js

```diff
@@ -0,0 +1,36 @@
+const express = require('express');
+const app = express();
+const array = [];
+
+app.listen(3000, () => {
+  // eslint-disable-next-line no-console
+  console.log('Express server listening on port 3000');
+});
+
+app.get('/api/grades', (req, res) => {
+  for (const x in grades) {
+    array.push(grades[x]);
+  }
+  res.send(array);
```

Comment:
Try moving your array into this callback function. As written now, your array will keep growing with every request.